app/main.py

```diff
@@ -1,15 +1,20 @@
 import streamlit as st
 
+# Set page configuration
+st.set_page_config(page_icon="assets\\favicon.ico", page_title='DeepMLeet')
```

Comment:
I like adding the page_title, however I have a logo I would rather use for this

Reply:
I added a template logo so you can replace with your own.